public/apps/configuration/panels/get-started.tsx

```diff
@@ -189,16 +187,7 @@ export function GetStarted(props: AppDependencies) {
           </EuiText>
 
           <EuiSpacer size="l" />
-          {props.config.ui.backend_configurable && (
-            <div className="text-center">
-              <EuiImage
-                size="xl"
-                alt="Three steps to set up your security"
-                url={securityStepsDiagram}
-                style={{ maxWidth: '100%', height: 'auto', width: 'auto' }}
-              />
-            </div>
-          )}
+          {props.config.ui.backend_configurable && <div className={"security-steps-diagram"} />}
```

Comment:
Please use single quotes here for `'security-steps-diagram'`. The failing checks are related to this.

Reply:
Thanks. Fixed. This repo should mimic the pre-commit hooks from OSD to avoid such questions from contributors.